crawler/config/centre.py

```diff
@@ -51,8 +54,8 @@
         "lab_id_default": "MK",
         "backups_folder": f"{CENTRE_DIR_BACKUPS}/MILK",
         "sftp_file_regex_unconsolidated_surveillance": f"^MK_{CENTRE_REGEX_SFTP_FILE_HERON}",
-        "sftp_file_regex_consolidated_surveillance": r"^RNA\d+\.csv$",
-        "sftp_file_regex_consolidated_eagle": r"^EGL\d+\.csv$",
+        "sftp_file_regex_consolidated_surveillance": r"^(RNA|cpRNA)\d+\.csv$",
```

Comment:
The cp RNA is constant here so I think you should try matching (cp) zero or 1 time.